libraries/provider_hab_package.rb

```diff
@@ -122,49 +119,44 @@ def depot_package(name, version = nil)
 
         def package_version(name, version = nil)
           p = depot_package(name, version)
-          unless p.nil?
-            i = p["ident"]
-            "#{i["version"]}/#{i["release"]}"
-          end
+          "#{p['ident']['version']}/#{p['ident']['release']}" unless p.nil?
         end
 
         def http
           # FIXME: use SimpleJSON when the depot mime-type is fixed
-          @http ||= Chef::HTTP::Simple.new("https://willem.habitat.sh/")
+          @http ||= Chef::HTTP::Simple.new('https://willem.habitat.sh/')
         end
 
-        def get_candidate_versions
+        def retrieve_candidate_versions
```

Comment:
The point of the rubocop warning for get_ methods is that this should just be candidate_versions